### PR TITLE
🔀🔖 4.7.3

### PR DESCRIPTION
**Pull request by jxn-30**
**Releasenotes**
* 🐛 [ecw/alarmTime] fix issue with vehicle tabs
* 🐛 [ecw/selectedVehicleCounter] fix text being unreadable in light mode
* 🐛 [em/complexes] fix issues with small buildings
* 🐛 [so,redesign/schoolings] fix countdown issues
* 🐛 [hotkeys] fix building hotkeys
* 🚸 [ecw/arrSearch] inline search to avoid jumps

---
**Releasenotes for translations**
```md
* 🐛 [$s('extendedCallWindow.alarmTime')] fix issue with vehicle tabs
* 🐛 [$s('extendedCallWindow.selectedVehicleCounter')] fix text being unreadable in light mode
* 🐛 [$s(!'extendedMap.buildingComplexes')] fix issues with small buildings
* 🐛 [$m('schoolingOverview')/, $m('redesign')/schoolings] fix countdown issues
* 🐛 [$m('hotkeys')] fix building hotkeys
* 🚸 [$s('extendedCallWindow.arrSearch')] inline search to avoid jumps
```

### CODE REVIEW

**Review by kdev — APPROVED** [review]

File: docs/.vuepress/components/bug-list.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
                         ></span>
                     </button>
                     <p class="bug-header-title">
-                        <a :href="bug.html_url" target="_blank">{{
+                        <a :href="bug?.html_url" target="_blank">{{
```
Comment (Cr4zyc4k3) [inline docs/.vuepress/components/bug-list.vue:19]: Why is the question mark necessary?

File: src/modules/extendedMap/components/buildingComplexes/buildingComplex.vue [hunk 2]
```diff
@@ -1725,31 +1725,24 @@ export default Vue.extend<
...
-                            (
-                                Object.entries(smallBuildings).find(
-                                    ([, small]) =>
-                                        small === building.building_type
-                                )?.[0] ?? bigBuildingType
-                            ).toString()
-                        );
-                    }
+
+                    const realBuildingType = building.small_building
```
Comment (Cr4zyc4k3) [inline src/modules/extendedMap/components/buildingComplexes/buildingComplex.vue:1729]: Please use at least brackets to maintain a little bit of readability.

File: docs/.vuepress/components/bug-list.vue (hunk 1 above)
Comment (jxn-30) [reply docs/.vuepress/components/bug-list.vue:19]: The build did fail sometimes due to `bug` being undefined and `bug.user` being undefined in lines 49&50.
I couldn't find out when and why that happened but this prevents the build from failing when that happens.

File: src/modules/extendedMap/components/buildingComplexes/buildingComplex.vue (hunk 2 above)
Comment (jxn-30) [reply src/modules/extendedMap/components/buildingComplexes/buildingComplex.vue:1729]: ESLint removes the brackets so I cannot make this section more readable by adding brackets.

Comment (Cr4zyc4k3) [reply src/modules/extendedMap/components/buildingComplexes/buildingComplex.vue:1729]: Do we really need Eslint? :wink:

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**